Support for gatsby 4?

I managed to install highlight code without issues; but after upgrading npm I now get this error:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: constant-solutions@1.0.0
npm ERR! Found: gatsby@4.4.0
npm ERR! node_modules/gatsby
npm ERR!   gatsby@"^4.4.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer gatsby@"^3.0.0" from gatsby-remark-highlight-code@3.1.0
npm ERR! node_modules/gatsby-remark-highlight-code
npm ERR!   gatsby-remark-highlight-code@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
```

**1: **
This generates an `ERR!` not just a `WARN`:

```
npm ERR! While resolving: highlight-sad@1.0.0
npm ERR! Found: gatsby@4.4.0
npm ERR! node_modules/gatsby
npm ERR!   gatsby@"^4.4.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer gatsby@"^3.0.0" from gatsby-remark-highlight-code@3.1.0
npm ERR! node_modules/gatsby-remark-highlight-code
npm ERR!   gatsby-remark-highlight-code@"^3.1.0" from the root project
```

**2:**
I was able to get my site up and running by adding an override to my package.json:

```
  "overrides": {
    "gatsby-remark-highlight-code": {
      "gatsby": "4.4.0"
    }
  }
```

That implies to me that there aren't any breaking changes, but I'm no Gatsby expert.


For anyone reading this bug: **This is a terrible idea.  Don't do this.  Wait until it's supported.**

**3:**
Please see https://github.com/DylanLacey/gatsby-remark-code-4 for a minimal repro; Just run `npm install`.

--- a/package.json
+++ b/package.json
@@ -13,7 +13,7 @@
     "unist-util-visit": "^2.0.2"
   },
   "peerDependencies": {
-    "gatsby": "^3.0.0",
+    "gatsby": "^3.0.0 || ^4.0.0",
     "@deckdeckgo/highlight-code": "^3.6.0"
   },
   "repository": {

